frontend/src/models/management/UserSubmissionInfo.ts

```diff
@@ -0,0 +1,15 @@
+export default class UserSubmissionInfo {
```

Comment:
Did not comment this in the backend, but guessing whether this is really necessary